src/pages/AuthenticationPages/Login.tsx

```diff
@@ -0,0 +1,150 @@
+import { Typography, Button } from "@mui/material"
+import { useState } from "react"
+import axios from "axios"
+import { useAppDispatch } from "../../redux/hooks"
+import { Link, useNavigate } from 'react-router-dom'
+import jwt_decode from "jwt-decode"
+import {
+    notify
+} from "../../App"
+import {
+    setUserAsLoggedIn
+} from "../../redux/index"
+import {
+    AiOutlineEye,
+    AiOutlineEyeInvisible
+} from "../../assets/react-icons"
+import './userAuth.css'
+
+const Login = () => {
+    const dispatch = useAppDispatch()
+    const navigate = useNavigate()
+
+    const [userEmail    , setUserEmail]    = useState('')
+    const [userPassword , setUserPassword] = useState('')
+    const [newPasswordType, setNewPasswordType] = useState('password')
+
+    function toggleNewPasswordAppearance()
+    {
+      if(newPasswordType==="password")
+      {
+        setNewPasswordType("text")
+      }
+      else
+      {
+        setNewPasswordType("password")
+      }
+    }
+
+    function loginUser(event: React.FormEvent)
```

Comment:
Suggestion:  You can move all the api calls into a services folder, and also move the loginUser in to maybe utils/authUtils